xbmc/filesystem/DiscDirectoryHelper.h

```diff
@@ -0,0 +1,112 @@
+/*
+ *  Copyright (C) 2005-2025 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#include <map>
+#include <string>
+#include <utility>
+#include <vector>
+
+static constexpr unsigned int MIN_EPISODE_DURATION = 10 * 60; // 10 minutes
+static constexpr unsigned int MAX_EPISODE_DIFFERENCE = 30; // 30 seconds
+static constexpr unsigned int MIN_SPECIAL_DURATION = 5 * 60; // 5 minutes
+static constexpr unsigned int MAIN_TITLE_LENGTH_PERCENT = 70;
+
+struct PlaylistInfo
+{
+  unsigned int duration{0};
+  std::vector<unsigned int> clips;
+  std::vector<unsigned int> chapters;
+  std::string languages;
+};
+
+struct ClipInfo
+{
+  unsigned int duration{0};
+  std::vector<unsigned int> playlists;
+};
+
+using PlaylistMap = std::map<unsigned int, PlaylistInfo>;
+using PlaylistMapEntry = std::pair<unsigned int, PlaylistInfo>;
+using PlaylistVector = std::vector<std::pair<unsigned int, PlaylistInfo>>;
+using PlaylistVectorEntry = std::pair<unsigned int, PlaylistInfo>;
+using ClipMap = std::map<unsigned int, ClipInfo>;
+
+enum class GetTitlesJob : uint8_t
```

Comment:
Type not used in this header. And `jobs` are definitely not related to the functionality of this helper class. If needed, the definition belongs elsewhere (another header, which actually defines details for `get titles job`).